src/workspace-data/data-table/uri-viewer/UriPreview.js

```diff
@@ -22,23 +22,24 @@ const styles = {
   },
 };
 
-const isImage = ({ contentType, name }) => {
-  return /^image/.test(contentType) || /\.(?:(jpe?g|png|svg|bmp))$/.test(name);
+export const isImage = ({ contentType, name }) => {
+  return /^image/.test(contentType) || /\.(jpe?g|png|svg|bmp)$/.test(name);
 };
 
-const isText = ({ contentType, name }) => {
-  return /(?:(^text|application\/json))/.test(contentType) || /\.(?:(txt|[ct]sv|log|json))$/.test(name);
+export const isText = ({ contentType, name }) => {
+  return /(^text|application\/json)/.test(contentType) || /\.(txt|[ct]sv|log|json|fastq|fasta|fa|vcf|sam|bed|interval_list|gtf|md|html)$/.test(name);
 };
 
-const isBinary = ({ contentType, name }) => {
-  // excluding json and google's default types (i.e. wasn't set to anything else)
-  return (
-    /application(?!\/(json|octet-stream|x-www-form-urlencoded)$)/.test(contentType) || /(?:(\.(?:(ba[mi]|cra[mi]|pac|sa|bwt|gz))$|\.gz\.))/.test(name)
-  );
+export const isBinary = ({ contentType, name }) => {
+  return /application(?!\/(json|octet-stream|x-www-form-urlencoded)$)/.test(contentType) || /(\.(ba[mi]|cra[mi]|bcf|h5ad|pdf|gz)$|\.gz\.)/.test(name);
```

Comment:
this change removed support for `pac`, `sa`, and `bwt` … is that intentional?

Reply:
I have added back `pac`, `sa`, and `bwt`